packages/gravity-ui-nunjucks/src/_patterns/03-organisms/00-page-structure/05-callout.njk

```diff
@@ -0,0 +1,10 @@
+{%- from 'molecules-three-column-list-justified' import threeColumnListJustified -%}
+
+<div class="grav-c-callout">
+  <img src="{{ backgroundImage }}" />
```

Comment:
Hmm.. I'm torn about this. Using an `img` element as you've done is fine and works. But, I wonder if using a background image here would be easier. @dw-buildit - do you have a preference?

That being said, _if_ we leave this as an `img`, then you must add an `alt=""` attribute to it. Alt attributes are mandatory but, if the image is purely decorative, it's value should be an empty string. See: https://www.w3.org/WAI/tutorials/images/decision-tree/

Reply:
I think the general rule i've used is use `<img>` for content images and `background-image:` for non content images. The main difference I think is that images get referenced by assistive tech (although you've mentioned how to correctly mark up decorative images) and they get printed. Whereas background images do not.

We don't have a print stylesheet so the printing question is not as clear cut. If we did I'd probably argue that this should be a background image so that we can print the text as standard black on white rather than having a large block print with white text within.

Reply:
I'm honestly fine with using this as an `<img>` or background image. @james-nash and @dw-buildit if you could let me know how to proceed, as I'm not sure how to resolve this atm.

Reply:
Considering both of @dw-buildit's comments, I propose the following:

- We completely remove images from this Gravity component. It would therefore only concern itself with laying out the big text and the 3 column list thing. This side-steps any potential a11y issues for whatever `@buildit/gravity-ui-web` provides out of the box
- Then, in the website's own CSS, code we customise the instance of this component on the homepage by adding the desired `background-image`.

My thinking is, the website project bears then responsibility for choosing an appropriate background image (and color scheme).

Sound good?

Reply:
Sounds good, I'll remove the background image then 👍